src/compiler/crystal/codegen/codegen.cr

```diff
@@ -613,6 +623,12 @@ module Crystal
 
       if return_phi = context.return_phi
         return_phi.add @last, node_type
+      elsif catch_pad = @catch_pad
```

Comment:
Yeah, I didn't really look at the spec suite since it's impossible to run the spec suite on windows yet because the compiler depends on a bunch of stuff thats not ported.